src/components/Hero.js

```diff
@@ -0,0 +1,33 @@
+import React from "react"
+import { Container, Row, Col } from "react-bootstrap"
+import heroHome from "../assets/images/heroHome.jpg"
+import { StaticImage } from "gatsby-plugin-image"
+import YohDevButton from "./YohDevButton"
+
+const sectionStyle = {
+  backgroundImage: `url(${heroHome})`,
+}
+
+const Hero = (props) => {
+  return (
+    <section className="hero frontpage" style={sectionStyle}>
+      <Container>
+        <Row className="align-items-md-center">
+          <Col lg={6} className="left">
+            <StaticImage
```

Comment:
Can this be passed as a prop? Import it into the home.js and passed. May need GatsbyImage instead of StaticImage.

Reply:
Left image is now dynamic